app/controllers/idv/how_to_verify_controller.rb

```diff
@@ -21,11 +22,15 @@ def update
       )
 
       if result.success?
-        if how_to_verify_form_params['selection'] == Idv::HowToVerifyForm::REMOTE
+        if how_to_verify_form_params['selection'] == 'remote'
```

Comment:
I don't feel strongly, but why are we moving away from the constant?

Reply:
The constants were removed during a long struggle to get the code working. We could probably put them back now that the code is working.

Reply:
Done. Resolved with commit fc6308d7804a87ec5c53f4b1eb1680960ec37e1c